src/hooks/http.ts

```diff
@@ -238,13 +306,22 @@ function handleResponse(
   startTime: number,
   timestamp: string | undefined,
   response: http.ServerResponse<http.IncomingMessage>,
+  chunks: Chunk[],
 ): void {
   if (fixupEvent(request, requestEvent)) recording.fixup(requestEvent);
+
+  let body =
+    chunks.length > 0 && typeof chunks[0] === "string"
+      ? chunks.join("")
+      : Buffer.concat(chunks as Buffer[]).toString("utf8");
+  body = body.substring(0, config.responseBodyMaxLength);
```

Comment:
In case of truncation it would be nice if it had `... (42 more characters)` rider to make it clear.